docker-compose.yml

```diff
@@ -0,0 +1,35 @@
+version: '3.8'
+services:
+  mysql:
+    image: mysql:8.0
+    container_name: mysql_db
+    restart: always
+    environment:
+      MYSQL_ROOT_PASSWORD: rootpassword
+      MYSQL_DATABASE: todolist
+      MYSQL_USER: user
+      MYSQL_PASSWORD: userpassword
+    volumes:
+      - mysql_data:/var/lib/mysql
+    ports:
+      - "3306:3306"
+
+  todolist:
+    build:
+      context: .
+    container_name: todolist_app
+    restart: always
+    environment:
+      DB_HOST: mysql
+      DB_PORT: 3306
+      DB_NAME: todolist
+      DB_USER: user
+      DB_PASSWORD: userpassword
+    depends_on:
+      - mysql
+    ports:
+      - "8000:8000"
```

Comment:
The port mapping for the Django application is set to `8000:8000`, but the instructions specify that it should be accessible on port `8080`. Consider changing this to `8080:8000` to match the instructions.